src/it/resources/org/checkstyle/suppressionxpathfilter/missingoverride/SuppressionXpathRegressionMissingOverrideAnonymous.java

```diff
@@ -0,0 +1,11 @@
+package org.checkstyle.suppressionxpathfilter.missingoverride;
+
+public class SuppressionXpathRegressionMissingOverrideAnonymous {
+    Runnable r = new Runnable() {
+
+        /** {@inheritDoc} */
```

Comment:
here javadoc is single-line, in other inputs it is multiline.
Is there some reason for it?

Reply:
sorry, no some reason, just a small oversight

Reply:
done